spec/cucumberConf.js

```diff
@@ -0,0 +1,35 @@
+var env = require('./environment');
```

Comment:
We do not provide first party cucumber support, and the `.feature` files referenced here do not exist